interfaces/cython/cantera/test/test_onedim.py

```diff
@@ -300,6 +300,16 @@ def test_soret_flag(self):
         self.sim.transport_model = 'Multi'
         self.sim.soret_enabled = True
 
+    def test_set_mix_with_soret(self):
+        """Test that setting mixture averaged transport with
+        Soret diffusion enabled raises an error.
+        """
```

Comment:
Don't ask me why, but `unittest` doesn't get along well with docstrings. See https://travis-ci.org/Cantera/cantera/jobs/349570469#L3048. The easiest thing to do is just write this as a comment.

Reply:
👍